local/kind/setup.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
```

Comment:
It would be great to have a teardown script as well so a user can easily clean-up the resources when they need to.

Reply:
with ctrl-c kcp, controller and envoy are stopped
files are in /tmp meaning that if nothing is done they are automatically cleaned up on reboot or after they have not been accessed for a time, which is configurable and different from OS to OS.
for kind, it is a single command:  kind delete clusters us-east1 us-west1

Reply:
I will add notes to the doc. For the 3 lines it may be as good as a script and will give to the user more flexibility